README.md

```diff
@@ -34,3 +34,19 @@ To see how it works just simply issue at your shell prompt:
 ```
 $ jlc-kicad-tools
 ```
+
+### KiCad
+
+1. Design your board as usual
+1. Create a custom field in Eeschema (for instance LCSCStockCode) and add in the components "C" stock number from JLCPCB/LCSC
+1. Once you are satisfied with your board design, go to Eeschema and run “Tools -> Generate Bill of Materials”
+1. Go to Pcbnew and run ‘File -> Fabrication Outputs -> Footprint Position (.pos) File’.
```

Comment:
Go to Pcbnew and run "File -> Fabrication Outputs -> Footprint Position (.pos) File".

To keep consistency.